pkg/builtins/pedersen.go

```diff
@@ -0,0 +1,87 @@
+package builtins
+
+import (
+	starknet_crypto "github.com/lambdaclass/cairo-vm.go/pkg/starknet_crypto"
+	"github.com/lambdaclass/cairo-vm.go/pkg/vm/memory"
+)
+
+const PEDERSEN_BUILTIN_NAME = "pedersen"
+const PEDERSEN_CELLS_PER_INSTANCE = 3
+
+type PedersenBuiltinRunner struct {
+	base               memory.Relocatable
+	included           bool
+	verified_addresses []bool
+}
+
+func NewPedersenBuiltinRunner() *PedersenBuiltinRunner {
+	return &PedersenBuiltinRunner{}
+}
+
+func (r *PedersenBuiltinRunner) Include(include bool) {
+	r.included = include
+}
+
+func (p *PedersenBuiltinRunner) Base() memory.Relocatable {
+	return p.base
+}
+
+func (p *PedersenBuiltinRunner) Name() string {
+	return PEDERSEN_BUILTIN_NAME
+}
+
+func (p *PedersenBuiltinRunner) InitializeSegments(segments *memory.MemorySegmentManager) {
+	p.base = segments.AddSegment()
+}
+
+func (p *PedersenBuiltinRunner) InitialStack() []memory.MaybeRelocatable {
+	if p.included {
+		return []memory.MaybeRelocatable{*memory.NewMaybeRelocatableRelocatable(p.base)}
+	} else {
+		return nil
+	}
+}
+
+func (p *PedersenBuiltinRunner) DeduceMemoryCell(address memory.Relocatable, mem *memory.Memory) (*memory.MaybeRelocatable, error) {
+	if address.Offset%PEDERSEN_CELLS_PER_INSTANCE != 2 || p.CheckVerifiedAddresses(address) {
```

Comment:
Lets add a constant PEDERSEN_INPUT_CELLS_PER_INSTANCE = 2 for more clarity

Reply:
Thanks fede!
DONE [189d045](https://github.com/lambdaclass/cairo-vm_in_go/pull/152/commits/189d045bcdba11e4cb760f0e68f1cd2c7577b70b)